### PR TITLE
Remove gRPC module from lang repository

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose
Remove gRPC module from lang repository

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: distribution/zip/jballerina-tools/build.gradle [hunk 1]
```diff
@@ -84,7 +84,6 @@ dependencies {
     dist 'com.fasterxml.jackson.core:jackson-core:2.8.6'
     dist 'com.fasterxml.jackson.core:jackson-annotations:2.8.0'
     dist 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.8.8'
-    dist 'com.google.api.grpc:proto-google-common-protos:1.17.0'
```
Comment (BuddhiWathsala) [inline distribution/zip/jballerina-tools/build.gradle:87]: I saw this dependency has been used in MySQL, Kafka, and oauth2. When we remove this here, it won't be available in the ballerina runtime. @niveathika, @ThisaruGuruge, and @ldclakmal, please note.

**Review by BuddhiWathsala — APPROVED** [review]

File: distribution/zip/jballerina-tools/build.gradle (hunk 1 above)
Comment (daneshk) [reply distribution/zip/jballerina-tools/build.gradle:87]: I don't think this library is used in other modules. AFAIK this is only used in protobuf tool. will revert if its not

Comment (ThisaruGuruge) [reply distribution/zip/jballerina-tools/build.gradle:87]: This is not used in Kafka.

Comment (ldclakmal) [reply distribution/zip/jballerina-tools/build.gradle:87]: This is not used in oauth2.

Comment (niveathika) [reply distribution/zip/jballerina-tools/build.gradle:87]: This is not used in MySQL too

**Pull request merged**